src/Orleans.Core.Abstractions/Orleans.Core.Abstractions.csproj

```diff
@@ -11,5 +11,10 @@
     <RootNamespace>Orleans</RootNamespace>
     <TargetFramework>netstandard1.3</TargetFramework>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.3.0" />
```

Comment:
Please use the $(SystemRuntimeVersion) variable, no baked in versions in project files.